tests/pyccel/test_pyccel.py

```diff
@@ -442,3 +439,6 @@ def test_multiple_results():
 #------------------------------------------------------------------------------
 def test_elemental():
     pyccel_test("scripts/decorators_elemental.py")
+
+def test_print(language):
+    pyccel_test("runtest_print.py", language=language, output_dtype=str)
```

Comment:
It is better to use a dtype for each line to avoid problems with the precision of floats and complexes. If you group the outputs by type then you can use `[str]*nlines` to avoid typing the same thing over and over (see `test_expressions`)

Reply:
yes, I am using now dtype. but I made some changes to how strings being compared.
I did this change to be able to test the Fortran print strings.